libraries/AC_AutoTune/AC_AutoTune_Multi.cpp

```diff
@@ -58,7 +59,13 @@
 #define AUTOTUNE_FLTE_MIN                  2.5     // minimum Rate Yaw error filter value
 #define AUTOTUNE_RP_MIN                   0.01     // minimum Rate P value
 #define AUTOTUNE_RP_MAX                    2.0     // maximum Rate P value
-#define AUTOTUNE_SP_MAX                   40.0     // maximum Stab P value
+
+#if APM_BUILD_TYPE(APM_BUILD_ArduPlane)
```

Comment:
I also feel a parameter for all vehicle types would be more appropriate here

Reply:
Adding a parameter is beyond the scope of this pr. If reducing the default for plane isn't acceptable I am happy to remove it.

As it is I added it because Tridge requested it. However if this is something that is going to get in the way of an important bug fix, we can leave that to a separate PR.